### ISSUE TITLE
the 'main' key in package.json doesn't point to the correct file

### ISSUE THREAD

**Issue by kjhangiani** [tnwinc/htmldiff.js#7]
I was interested in testing this module out in an ember project using ember-browserify, but was running into errors because it seems that the `main: "htmldiff.js"` entry point in package.json is incorrect.  Manually changing this to `main: "./src/htmldiff.js"` allows the module to work.

Is this a bug in package.json? Or should there be an htmldiff.js in the root level of the node_modules/htmldiff folder?  When I npm install this package, htmldiff.js and htmldiff.coffee are both only in the src/ folder

**Comment by sant123** [comment]
Yes it is a bug, you have to manually change the main config in the package json. Also try using [json](https://www.npmjs.com/package/json) package and create an script that edits the package.json on installing packages.

**Comment by ITsolution-git** [comment]
Could you please fix this?
